web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisCreate.tsx

```diff
@@ -20,6 +20,7 @@ import { AxiosError, AxiosResponse } from 'axios'
 export const AnalysisCreate = (): JSX.Element => {
   const { asset_id } = useParams<{ asset_id: string }>()
   const settings = useContext(ApplicationContext)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore